enhancements/monitoring/replace-prometheus-adapter-with-metrics-server.md

```diff
@@ -0,0 +1,202 @@
+---
+title: replace-prometheus-adapter-with-metrics-server
+authors:
+  - "@slashpai"
+reviewers:
+  - "@simonpasquier"
+  - "@dgrisonnet"
+  - "@openshift/openshift-team-monitoring"
+approvers: 
+  - "@simonpasquier"
+  - "@dgrisonnet"
+  - "@openshift/openshift-team-monitoring"
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-06-20
+last-updated: 2023-06-20
+tracking-link:
+  - https://issues.redhat.com/browse/MON-3153
+---
+
+# Replace prometheus-adapter with metrics-server
+
+## Summary
+
+This proposal suggests replacing the [Prometheus Adapter](https://github.com/kubernetes-sigs/prometheus-adapter) currently used for monitoring resource metrics in OpenShift with the [Metrics Server](https://github.com/kubernetes-sigs/metrics-server).
+
+## Motivation
+
+For autoscaling OpenShift, a resource metrics implementation is necessary. Currently, this is dependent on cluster monitoring operator(CMO) managed Prometheus stack.
+
+Reasons to consider to moving to Metrics Server are:
+
+1. **Upcoming Archival of prometheus-adapter:** The Prometheus Adapter project will soon be archived due to difficulties in maintaining the project, making it less desirable for long-term usage.
+
+1. **Simplified Monitoring Architecture:** By replacing Prometheus Adapter with Metrics Server, we can reduce the complexity and maintenance overhead of the monitoring system in OpenShift since `metrics-server` does not rely on Prometheus.
+
+1. **Lightweight Solution:** Deploying and configuring Prometheus Adapter requires additional effort compared to Metrics Server. Prometheus Adapter is tightly coupled to Prometheus and this creates a bottleneck for allowing monitoring as  optional feature. See [MON-3152](https://issues.redhat.com/browse/MON-3152).
+
```

Comment:
can we add "better accuracy" (by default metrics-server will take into account the sample timestamps) and "increased robustness" (one only component not depending on an external service)?